File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    width, length = [], []
    for i in range(len(sizes)):
        if sizes[i][0] < sizes[i][1]:
            sizes[i][0], sizes[i][1] = sizes[i][1], sizes[i][0] 
        width.append(sizes[i][0])
        length.append(sizes[i][1])
        
    answer = max(width) * max(length)
    return answer

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int h = 0, w = 0;
        for(int[] s : sizes){
            h = Math.max(h, Math.max(s[0], s[1]));
            w = Math.max(w, Math.min(s[0], s[1]));
        }
        return h * w;
    }
}